apps/meteor/client/views/room/contextualBar/ExportMessages/MailExportForm.tsx

```diff
@@ -97,7 +97,7 @@ const MailExportForm: FC<MailExportFormProps> = ({ onCancel, rid }) => {
 	const roomsExport = useEndpoint('POST', '/v1/rooms.export');
 
 	const handleSubmit = async (): Promise<void> => {
-		if (toUsers.length === 0 && additionalEmails === '') {
+		if (!toUsers || (toUsers.length === 0 && !additionalEmails)) {
```

Comment:
This conditional feels wrong. Should it really have a different behavior when `toUsers` is not set than when it is set to an empty array?
If that's correct, please add a comment explaining the reason.

Reply:
Actually, it's never going to be `undefined.`